### PR TITLE
switch to GitHub actions

### PR DESCRIPTION
**Pull request by dlockhart**
As per discussion with Maya, I've switched this to use semantic-release for automatic release. I've updated the README with instructions on how to do that.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,5 @@
 # foundation-components
 
-[![NPM version](https://img.shields.io/npm/v/@brightspace-ui/polaris.svg)](https://www.npmjs.org/package/@brightspace-ui/polaris)
-[![Dependabot badge](https://flat.badgen.net/dependabot/BrightspaceUI/polaris?icon=dependabot)](https://app.dependabot.com/)
-[![Build status](https://travis-ci.com/@brightspace-ui/polaris.svg?branch=master)](https://travis-ci.com/@brightspace-ui/polaris)
```
Comment (dlockhart) [inline README.md:5]: This was never deployed to NPM, so the badge was broken. As was the Dependabot one, which GitHub broke and hasn't fixed yet.

**Review by m6jones — APPROVED** [review]
Super amazing!!!

File: components/activity/editor/collection/d2l-activity-editor-main-collection.js [hunk 2]
```diff
@@ -65,7 +65,7 @@ class ActivityEditorMainCollection extends HypermediaStateMixin(LitElement) {
 				</div>
 				<div class="d2l-activity-collection-activities">
 					<d2l-list @d2l-list-item-position-change="${this._moveItems}">
-						${repeat(this.items, href => href, href => html`<d2l-activity-list-item href="${href}" .token="${this.token}" draggable key="${href}"></d2l-activity-item>`)}
+						${repeat(this.items, href => href, href => html`<d2l-activity-list-item href="${href}" .token="${this.token}" draggable key="${href}"></d2l-activity-list-item>`)}
```
Comment (dlockhart) [inline components/activity/editor/collection/d2l-activity-editor-main-collection.js:68]: Looks like this lint error was failing in Travis for some time as well.

Comment (m6jones) [reply components/activity/editor/collection/d2l-activity-editor-main-collection.js:68]: Yeah, ignore that. This repo is still in POC land.

**Review by svanherk — APPROVED** [review]

**Pull request merged**